include/boost/geometry/core/config.hpp

```diff
@@ -17,6 +18,8 @@
 #define BOOST_GEOMETRY_CXX11_ARRAY_UNIFIED_INITIALIZATION
 #endif
 
+#define BOOST_GEOMETRY_USE_KRAMER_RULE
```

Comment:
I wonder, is this a public macro that a user is supposed to (un)define to tweak the behaviour here?

If public, is this documented anywhere?

Reply:
It seems that this macro is not used any more and used once to bypass the validity checks on some test cases in union, but @barendgehrels should be the right person to answer.

Reply:
Sorry for the delayed answer.

The macro is transitional, in the time to remove the rescaling. It selects or deselects an algorithm. It is not meant to the public (though they might use it). I will come back to it later.

For now I need it because some testcases depend on its value. Therefore it is part of this PR.

I will write a short comment.

Reply:
@barendgehrels Thanks for the explanation.